### PR TITLE
Features/2.0

### PR DESCRIPTION
**Pull request by bstiq**
TODO

- [x] ESD protection 
- [x] switch flash memory to C190862
- [x] per key rgb pcb
- [x] lcd screen, add pinout
- [x] update swd pinout
- [x] usb c cutout  - main pcb
- [x] mark non assembly components
- [x] add IO pads on other side as well
- [x] make ribbon cable hole slightly bigger on the outer (opposite audio jack)
- [x] fix lcd screen wiring
- [x] add back footprint for optional big reset button
- [ ] ~~make encoder holes plated // fuse with switch holes~~
- [ ] ~~add more screw holes~~

TODO BOM

- [x] check decoupling capacitors and resistors: value, size (1uf/330 == change resistor)
- [ ] ~~remove big reset button from BOM, or check if JLC parts has it~~

TODO silkscreen
- [x] version number
- [x] IO on other side
- [x] reset button
- [x] led detrompeur
- [ ] ~~add column and row numbers on switches~~

TODO with the rgb:
- [x] add holes for the rotary encoder pass through (thumb)
- [x] add fpc connector, make sure there's space on both keyboards
- [x] add capacitor and resistor
- [ ] ~~add more underglow rgb~~
- [x] add sk6812 footprints + maybe pre assembled 
- [ ] ~~add per key rgb signal output to connect other stuff~~
- [x] add underglow rgb output to connect stuff and bypass per key rgb

TODO

- [x] connect pin enc 1 and 2
- [x] remove big rst button, add label on small splinky button
- [x] breakout all available rp2040 pins
- [x] breakout vcc, gnd, rst
- [x] bigger hole for trackpad ribbon cable

TODO - RGB

- [x] switch to only one pin that goes to underglow, then per key RGB
- [x] find a way to short directly to per key if underglow stops working ?
- [x] per key rgb - make switch pin passthrough holes slightly bigger

OTHERS

- ~~[ ] bom: remove big rst button (and build guides)~~

OPTIONAL

- [x] bigger flash memory

Example of lcd screen:
https://de.aliexpress.com/item/1005004190019659.html?srcSns=sns_Copy&spreadType=socialShare&bizType=ProductDetail&social_params=20648406344&aff_fcid=11c5b20a72ff499cb399305abf67f1aa-1663576247153-06855-_mrMM7Sc&tt=MG&aff_fsk=_mrMM7Sc&aff_platform=default&sk=_mrMM7Sc&aff_trace_key=11c5b20a72ff499cb399305abf67f1aa-1663576247153-06855-_mrMM7Sc&shareId=20648406344&businessType=ProductDetail&platform=AE&terminal_id=4ce21b16a3d84d1a9b46ab7e9a192d35&afSmartRedirect=y

### CODE REVIEW

**Comment by bstiq** [comment]
Getting ready for order !

**Comment by bstiq** [comment]
Feedback from @0xcharly :

The manually installed reset button adds potential for user error.
Removing it makes inventory simpler, and build support/help faster.

**Comment by bstiq** [comment]
Instead of breaking out all the rp2040 pins, only 2 of them (easy enough to route, should be enough :tm: )

**Comment by bstiq** [comment]
Update from charly : hole for trackpad cable should be a little bit more towards center, and bigger
image

**Comment by bstiq** [comment]
Note: For the per key RGB it may be possible to use the SK6803 mini e leds

File: README.md [hunk 1]
```diff
@@ -143,8 +149,59 @@ If building the DIY version, you will also need:
...
+
+If you are using the acrylic midplate, you can install a full bottom plate. Order the [bottom acrylic plates](mechanical/plates/v2/bottom_acry_full.dxf) in 3mm thickness.
 
-### 3d printed thin case
+You will also need to chamfer the M3 holes for the screws to be flush.
+
+
+### Tents - metal bottom plate
+
+This plate enables the use of tripods through magsafe magents.
```
Comment (ykz89) [inline README.md]:
```suggestion
This plate enables the use of tripods through magsafe magnets.
```

**Pull request merged**